packages/common/src/ui/icons/BarChart.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon';
+
+export const BarChartIcon = ({ ...props }: SvgIconProps): JSX.Element => {
```

Comment:
have hard coded colours for this, and I feel bad about that. But Manu's version has three different colours 🤷 
could use secondary / primary + light, main and dark variants - but we've used different versions of the secondary variants already. I think I'll update in the next PR anyway :)

Reply:
Also I don't think it's too bad to hard code some colors in specific places

Possibly having the `getPaletteColour` just return a tuple of three colours depending on what colour was passed `primary` or `secondary` (And then throw an error if not primary or secondary saying unsupported 🤔  .. or just support primary..) 🤷

Reply:
nice, thanks - have done.